wgengine/magicsock/magicsock.go

```diff
@@ -2420,8 +2430,11 @@ func (c *connBind) Open(ignoredPort uint16) ([]conn.ReceiveFunc, uint16, error)
 	}
 	c.closed = false
 	fns := []conn.ReceiveFunc{c.receiveIPv4, c.receiveDERP}
+	health.ReceiveIPv4.Open()
```

Comment:
with Open setting both started and running, you're assuming that wireguard-go actually is going to launch it now.

I think I'd rather continue to just track intent here (we want it running) but track reality elsewhere like before, but with some grace period before reporting the error. Like 1-5 seconds to get from want-running to actually-running.

This is fine for now, though.